meta-lmp-base/recipes-core/ovmf/ovmf_git.bbappend

```diff
@@ -1,9 +1,2 @@
-FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
```

Comment:
there is a typo on this commit log

pacthes/patches

Reply:
thanks, I will fix it